referrer-policy/generic/fetch.js

```diff
@@ -0,0 +1,337 @@
+// Shamelessly copied from: https://github.com/github/fetch/blob/master/fetch.js
+// TODO(krisitijanburnik): See how to include third_party for WPT.
```

Comment:
I'm not sure it's a good idea to polyfill fetch in the tests. You'll be testing the polyfill rather than the feature in some subset of browsers. Did you consider simply PASSing the tests if `!window.fetch`?